x-pack/plugins/fleet/public/applications/fleet/mock/epm/details_api.ts

```diff
@@ -0,0 +1,539 @@
+          streams: [
+            {
+              input: 'logfile',
+              vars: [
+                {
+                  name: 'paths',
+                  type: 'text',
+                  title: 'Paths',
+                  multi: true,
+                  required: true,
+                  show_user: true,
+                  default: ['/var/log/nginx/error.log*'],
+                },
+              ],
+              template_path: 'stream.yml.hbs',
+              title: 'Nginx error logs',
+              description: 'Collect Nginx error logs',
+              enabled: true,
+            },
+          ],
+          package: 'nginx',
+          path: 'error',
+        },
+        {
+          type: 'logs',
+          dataset: 'nginx.ingress_controller',
+          title: 'Nginx ingress_controller logs',
+          release: 'experimental',
+          ingest_pipeline: 'default',
+          streams: [
+            {
+              input: 'logfile',
+              vars: [
+                {
+                  name: 'paths',
+                  type: 'text',
+                  title: 'Paths',
+                  multi: true,
+                  required: true,
+                  show_user: true,
+                  default: ['/var/log/nginx/ingress.log*'],
+                },
+              ],
+              template_path: 'stream.yml.hbs',
+              title: 'Nginx ingress controller logs',
+              description: 'Collect Nginx ingress controller logs',
+              enabled: false,
+            },
+          ],
+          package: 'nginx',
+          path: 'ingress_controller',
+        },
+        {
+          type: 'metrics',
+          dataset: 'nginx.stubstatus',
+          title: 'Nginx stubstatus metrics',
+          release: 'experimental',
+          streams: [
+            {
+              input: 'nginx/metrics',
+              vars: [
+                {
+                  name: 'period',
+                  type: 'text',
+                  title: 'Period',
+                  multi: false,
+                  required: true,
+                  show_user: true,
+                  default: '10s',
+                },
+                {
+                  name: 'server_status_path',
+                  type: 'text',
+                  title: 'Server Status Path',
+                  multi: false,
+                  required: true,
+                  show_user: false,
+                  default: '/nginx_status',
+                },
+              ],
+              template_path: 'stream.yml.hbs',
+              title: 'Nginx stub status metrics',
+              description: 'Collect Nginx stub status metrics',
+              enabled: true,
+            },
+          ],
+          package: 'nginx',
+          path: 'stubstatus',
+        },
+      ],
+      owner: { github: 'elastic/integrations-services' },
+      latestVersion: '0.3.7',
+      removable: true,
+      status: 'installed',
+    },
+  } as GetInfoResponse);
```

Comment:
I'd really like to avoid casting. Can you describe what's failing?

Reply:
I removed this casting after fixing a few things